Mantle/MTLJSONAdapter.m

```diff
@@ -370,13 +370,9 @@ + (NSDictionary *)valueTransformersForModelClass:(Class)modelClass {
 	for (NSString *key in [modelClass propertyKeys]) {
 		SEL selector = MTLSelectorWithKeyPattern(key, "JSONTransformer");
 		if ([modelClass respondsToSelector:selector]) {
-			NSInvocation *invocation = [NSInvocation invocationWithMethodSignature:[modelClass methodSignatureForSelector:selector]];
-			invocation.target = modelClass;
-			invocation.selector = selector;
-			[invocation invoke];
-
-			__unsafe_unretained id transformer = nil;
-			[invocation getReturnValue:&transformer];
+			IMP imp = [modelClass methodForSelector:selector];
+			NSValueTransformer*(*function)(id, SEL) = (void *)imp;
+			__unsafe_unretained NSValueTransformer *transformer = function(modelClass, selector);
```

Comment:
This attribute (or `__autoreleasing`) probably belongs on the function pointer's return type. This copy should be `__strong`.